rj_gameplay/rj_gameplay/role/runner_role.py

```diff
@@ -0,0 +1,253 @@
+import numpy as np
+import stp
+from rj_msgs.msg import RobotIntent
+
+# TODO: settle on unified way to define constants in gameplay
+from stp.utils.constants import RobotConstants  # , BallConstants
+
+from rj_gameplay.skill import line_kick, move, pivot_kick, receive
+
+# import stp.global_parameters as global_parameters
+# from stp.local_parameters import Param
+
+
+# TODO: move to constants file
+MIN_WALL_RAD = 0
+GOALIE_PCT_TO_BALL = 0.15
+DIST_TO_FAST_KICK = 7
+
```

Comment:
When you are working on a future project for RC, make sure to delete the commented out code unless it's something important (ie something to implement in the future).